json/Item_Weapon_JetBoots.txt

```diff
@@ -88,42 +88,42 @@
 		"jp_text": "ラムダレザスイフト",
 		"tr_text": "Lambda Lezaswift",
 		"jp_explain": "新素材で開発された魔装脚の改良型。\n安定性を重視し、再設計された。衝撃が\n少なく、軸がぶれにくいのが特徴。",
-		"tr_explain": ""
+		"tr_explain": "High-tech, improved Jet Boots. Re-designed to improve \nstability. Characterized by slightly shifted center \nand few crashes."
 	},
 	{
 		"assign": "116019",
 		"jp_text": "ディオレザスイフト",
 		"tr_text": "Dio Lezaswift",
 		"jp_explain": "新素材で開発された魔装脚の最終形。\n構造を変えることで、さらに軽量化。\n使用時の負担を大幅に軽減させた。",
-		"tr_explain": ""
+		"tr_explain": "Latest high-tech Jet Boots. \nTheir structure was improved again to make them \neven lighter, drastically reducing load."
 	},
 	{
 		"assign": "116020",
 		"jp_text": "レザスイフト",
 		"tr_text": "Lezaswift",
 		"jp_explain": "新素材を使用し開発された魔装脚。\n密度の高いフォトン刃が二枚付いており\n基礎モデルと比べ、軽量化されている。",
-		"tr_explain": ""
+		"tr_explain": "High-tech Jet Boots. Made much lighter by addition \nof two high-density Photon blades."
 	},
 	{
 		"assign": "116021",
 		"jp_text": "ラウンジブル",
 		"tr_text": "Roundibul",
 		"jp_explain": "動きを重視した結果、特殊な形状に\nなった魔装脚。重量による負担も\n大きいが、もたらす法撃力は高い。",
-		"tr_explain": ""
+		"tr_explain": "Jet Boots with unique shape that ephasizes movement. \nTheir propulsion was increased to make up for increased weight."
 	},
 	{
 		"assign": "116022",
 		"jp_text": "ディオラウンジブル",
 		"tr_text": "Dio Roundibul",
 		"jp_explain": "動き重視のため特殊な形状になった\n魔装脚の最終形。新素材により\n出力が増し、性能が向上した。",
-		"tr_explain": ""
+		"tr_explain": "Latest model of Jet Boots that emphasize movement. \nTheir output and performance was increased with \nhigh tech materials."
 	},
 	{
 		"assign": "116023",
 		"jp_text": "ラムダラウンジブル",
 		"tr_text": "Lambda Roundibul",
 		"jp_explain": "動き重視のため特殊な形状になった\n魔装脚の改良型。内部機構を改良し\n法撃力を高めることに成功した。",
-		"tr_explain": ""
+		"tr_explain": "Improved Jet Boots that emphasize movement. \nTheir internal mechanism was improved to increase propulsion."
```

Comment:
Line breaks need rearranging but otherwise perfect!
```suggestion
		"tr_explain": "Improved Jet Boots that emphasize\nmovement. Their internal mechanism\nwas improved to increase propulsion."
```